common/src/main/java/org/opensearch/sql/common/setting/Settings.java

```diff
@@ -23,6 +23,7 @@ public enum Key {
     SQL_SLOWLOG("plugins.sql.slowlog"),
     SQL_CURSOR_KEEP_ALIVE("plugins.sql.cursor.keep_alive"),
     SQL_DELETE_ENABLED("plugins.sql.delete.enabled"),
+    SQL_PAGINATION_API_SEARCH_AFTER("plugins.sql.pagination.api"),
```

Comment:
So we're not switching to PIT completely but support both PIT and Scroll?

Reply:
Current plan is to add for all pagination then remove scroll after performance comparison.

Reply:
So we will raise another PR for this after performance test?

Reply:
yes I was planning to add search_after for all queries (join, multi query, pagination) then a separate PR to remove scroll after performance test.

Reply:
Current default setting is true for using PIT with search after. After benchmark, we can decide if this needs to be changed. We can remove this while removing scroll.